### DISCUSSION TITLE
Roadrunner tutorial/repo?

### DISCUSSION THREAD

**Discussion by ltdangle** [roadrunner-server/roadrunner#971] [Q&A]
Hi guys,
I've been trying to setup Roadrunner for a while now (Mac OS and Ubuntu via Vagrant) but to no avail.
I couldn't find any good step-by-step tutorial either...

Perhaps you have a step-by-step tutorial that helped you which you can share?

Thank you!

**Comment by rustatian** [comment]
Hey @ltdangle, have you read the https://roadrunner.dev?

**Reply by ltdangle** [reply]
Yes, it didn't help :-(
For example, https://roadrunner.dev/docs/intro-config lists a zillion options that I'm lost in, I just want to be able to get the simplest php script running to test the water and see the advertised 'no bootstrap' performance boost.

**Reply by rustatian** [reply]
Here you are: https://roadrunner.dev/docs/php-worker

**Reply by rustatian** [reply]
If you have any questions, regarding the setup, creating worker, feel free to join our discord server (ru/en): https://discord.gg/TFeEmCs

**Reply by ltdangle** [reply]
> Here you are: https://roadrunner.dev/docs/php-worker

This helped, I was able to run the script according to instructions, thank you!

**Comment by ltdangle** [comment]
Thanks, @rustatian! Going to try that now...
